main_app/templates/main_app/login.html

```diff
@@ -3,31 +3,33 @@
 {% load static %}
```

Comment:
@coding-geek21 On the` login` page, the form and footer are now not overlapping but the position of both footer and form is distorted on the small screen. Can you please fix it?